### PR TITLE
Contributor guide review

### PR DESCRIPTION
**Pull request by clebergnu**
This is part of https://github.com/avocado-framework/avocado/issues/5757.

### CODE REVIEW

File: optional_plugins/spawner_remote/setup.py [hunk 1]
```diff
@@ -34,6 +34,8 @@
     name="avocado-framework-plugin-spawner-remote",
     version=VERSION,
     description="Remote (host) based spawner",
+    long_description="Remote (host) based spawner",
```
Comment (richtja) [inline optional_plugins/spawner_remote/setup.py:37]: IMO this is related to #5839

File: docs/source/guides/contributor/chapters/environment.rst [hunk 2]
```diff
@@ -86,3 +84,28 @@ variable ``$AVOCADO_EXTERNAL_PLUGINS_PATH``. The workflow could be::
...
+
+GPG Signatures
+--------------
+
+This is an optional step for most contributors, but if you're
+interested in ensuring that your contribution is linked to yourself,
+this is the best way to do so.
+
+To get a GPG signature, you can find many howtos on the internet, but
+it generally it works like this::
```
Comment (richtja) [inline docs/source/guides/contributor/chapters/environment.rst:96]: typo `it generally it works`/`it generally works`

File: docs/source/guides/contributor/chapters/styleguides.rst [hunk 3]
```diff
@@ -19,41 +19,20 @@ motivation, approach and effects::
 Signing commits
 ~~~~~~~~~~~~~~~
 
-Optionally you can sign the commits using GPG signatures. Doing
-it is simple and it helps from unauthorized code being merged without notice.
+If you've set a GPG signature, it's a good idea to put it use when
```
Comment (richtja) [inline docs/source/guides/contributor/chapters/styleguides.rst:22]: typo `put it use`/`put it in use`

File: docs/source/guides/contributor/chapters/styleguides.rst [hunk 4]
```diff
@@ -37,4 +37,9 @@ And if you are merging branches::
 Code style guide
 ----------------
 
-.. warning:: TODO: Add the Code Style Guide.
+Avocado uses the Black code style checker, and thus, you should follow
+its very opnionated style.  In reality, it's recommended to use your
```
Comment (richtja) [inline docs/source/guides/contributor/chapters/styleguides.rst:41]: typo `opnionated`/`opinionated`

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @clebergnu, thank you for this refactoring. Overall it LGTM, I just found few typos and can you please do the rebase to #5839, thank you.

**Comment by clebergnu** [comment]
Thanks for the review @richtja, I've addressed all the issues you've found.

File: docs/source/guides/contributor/chapters/environment.rst [hunk 5]
```diff
@@ -20,6 +40,9 @@ Or if you already have pip installed, you can run directly::
 
  $ pip install -r requirements-dev.txt
 
+If you intend to build the documentation location, please also run::
```
Comment (richtja) [inline docs/source/guides/contributor/chapters/environment.rst:43]: Didn't you mean locally instead of location?

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @clebergnu thank you for your updates. Unfortunately, I think I found one more typo.

File: docs/source/guides/contributor/chapters/environment.rst (hunk 5 above)
Comment (clebergnu) [reply docs/source/guides/contributor/chapters/environment.rst]: Yes, I did mean "locally".  Thanks for catching that!  I've sent a version with that typo fixed.

**Review by richtja — APPROVED** [review]
LGTM, thanks.

**Pull request merged**